src/renderer/components/workspace.js

```diff
@@ -43,6 +44,7 @@ const Workspace = ({
                 <NoArchiveSelected />
               </When>
               <Otherwise>
+                <ArchiveSearch />
```

Comment:
Instead of listening to ipc actions direction in the component, use `getUIState` and `setUIState` action and selector to toggle between search states, and pass that value here so you can display the component based on that value: `{searchVisible && <ArchiveSearch />` for example. And in `renderer/index.js` you would listen to the ipc:

```js
ipc.on('open-archive-search', () => {
  store.dispatch(setUIState('searchVisible', true));
});
```

Reply:
Ok I've pushed my update :) Now you can control the result with arrow up/down and enter key.